src/Nancy/HttpLink.cs

```diff
@@ -0,0 +1,283 @@
+namespace Nancy
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using Nancy.Responses.Negotiation;
+
+    /// <summary>
+    /// Represents one of possibly many RFC 5988 HTTP Links contained in a <see cref="HttpLinkBuilder"/>.
+    /// </summary>
+    public class HttpLink : IEquatable<HttpLink>
+    {
+        private static readonly HttpLinkParameterComparer parameterComparer = new HttpLinkParameterComparer();

```

Comment:
We'll probably just let this slip, but I'm curious... Didn't R# suggest this name to be `ParameterComparer`? If not, we might look into the DotSettings here.